src/app/app.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [RouterOutlet],
```

Comment:
issue

          styles: ['./app.component.scss']

          Explantion: The correct property for specifying styles in Angular components is 'styles' instead of 'styleUrl'.